FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Mutual_Auth/demo_config.h

```diff
@@ -143,6 +143,13 @@
  * #define democonfigCLIENT_PRIVATE_KEY_PEM    "...insert here..."
  */
 
+/**
+ * @brief For AWS IoT MQTT broker, the appropriate ALPN and SNI configurations
+ *  are passed to the demo when this macro is defined.
+ *
+ * #define democonfigUSE_AWS_IOT_CORE_BROKER
```

Comment:
It may be good to assign a value for the config instead of just leaving it empty
```suggestion
 * #define democonfigUSE_AWS_IOT_CORE_BROKER     (1)
```